indico/modules/rb/controllers/user/rooms.py

```diff
@@ -14,13 +14,15 @@
 # You should have received a copy of the GNU General Public License
 # along with Indico; if not, see <http://www.gnu.org/licenses/>.
 
+import re
 from datetime import date, datetime, timedelta, time
 
 from dateutil.relativedelta import relativedelta
 from flask import request, session
-from sqlalchemy import func
+from sqlalchemy import func, asc
```

Comment:
no need for `asc(...)`, it's the default for any kind of sorting and you can also call the `.asc()` or `.desc()` methods on column objects etc